plugins/ad9371.c

```diff
@@ -966,6 +973,27 @@ static int compare_gain(const char *a, const char *b)
 		return 0;
 }
 
+static void int_dec_freq_update(void)
+{
+	struct iio_channel *ch;
+	double freq;
+
+	ch = iio_device_find_channel(cap, "voltage0_i", false);
+	iio_channel_attr_read_double(ch, "sampling_frequency", &freq);
+
+	gtk_label_set_text(GTK_LABEL(label_sampling_freq_rx), g_strdup_printf ("%f", freq / mhz_scale));
+
+	ch = iio_device_find_channel(dds, "voltage0", true);
+	iio_channel_attr_read_double(ch, "sampling_frequency", &freq);
+	gtk_label_set_text(GTK_LABEL(label_sampling_freq_tx), g_strdup_printf ("%f", freq / mhz_scale));
+}
+
+static void int_dec_update_cb(GtkComboBox *cmb, gpointer data)
```

Comment:
This could use a better naming (there are many controls that have their values incremented/decremented). One suggestion: 'fpga_frequency_changed_cb'

Reply:
the name comes from interpolation decimation -> int_dec it was a copy paste from pluto but will go with the naming suggetion.